snapcraft/plugins/rust.py

```diff
@@ -105,9 +106,19 @@ def _build_env(self):
         env = os.environ.copy()
         env.update({"RUSTC": self._rustc,
                     "RUSTDOC": self._rustdoc,
-                    "RUST_PATH": self._rustlib})
+                    "RUST_PATH": self._rustlib,
+                    'RUSTFLAGS': self._rustflags()})
         return env
 
+    def _rustflags(self):
+        ldflags = shell_utils.getenv('LDFLAGS')
+        rustldflags = ''
+        flags = set(ldflags.split(' '))
+        flags.remove('')
```

Comment:
I will reset instead of revert as you seemed to use your gmail email and that email has not been used to sign the CLA